Samples/ResourceManagement/cs-wpf/wpf_packaged_app/wpf_packaged_app.csproj

```diff
@@ -10,8 +10,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.20348.19" />
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0-preview1" />
+    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22499-preview" />
```

Comment:
>10.0.22499-preview [](http://example.com/codeflow?start=74&length=18)

was this necessary, Erik? the upgrade, i mean

Reply:
Yes and no. It is necessary because it seems the WAS stable release has the SDK.BuildTools with a minimal version of 10.0.22000 as dependency. 
And no because I think we might not want to use the preview version, so I I should probably change that to 22000 :)